### ISSUE TITLE
Grindstone Cannot Recognize Items 

### ISSUE THREAD

**Issue by Kyrocakex** [TheBusyBiscuit/ExoticGarden#210]
## Description (Required)
 I just recently tested and ran the DEV version of slimefun with inventory pets and exotic gardens, however whenever I try using the grindstone (on items that I KNOW are unlocked and available), it says "Sorry, I could not recognize the item in my dispenser. Please put something in that I know." 

I tried relocatting the item to various different places in the dispenser (so in this case wheat, since I was trying to make wheat flour).

Keep in mind most of these items are disabled, the server owner I was editing it for needs to change the permissions on most items before they can be fully added to the server. Right now only Exotic Gardens, machines for said addon, Inventory pets and other food items are available for public use. 

Wanted to quickly say thank you so much for making this plugin, I cannot express my gratitude and how happy I am to have someone not only make a plugin as amazing as this, but be actively working on it. Thank you so much, and hopefully this doesn't make you all too stressed..It could very well be my own fault! I'm new to dev work. :)

## Steps to reproduce the Issue (Required)

![image](https://user-images.githubusercontent.com/89564741/131008816-b5032bbe-3f11-441d-8745-592720fa6692.png)

![image](https://user-images.githubusercontent.com/89564741/131008877-5b565ea0-b917-44c0-b475-8264ce3f64da.png)

## Expected behavior (Required)
 I expected it to grind wheat to make wheat flour. 

## Environment (Required)
 

 - Minecraft Version: 1.17.1
 - CS-CoreLib Version: Java 16
 - Slimefun Version: RC - 26 (git 6aba414a)
 - ExoticGarden Version: vDEV - 76 

![image](https://user-images.githubusercontent.com/89564741/131009455-4b66b2ae-b181-413e-98d8-70a265d3fb2c.png)

**Comment by Boomer-1** [comment]
did you put it in the correct spot? according to the guide it is the upper left

**Comment by Kyrocakex** [comment]
> It's actually working fine in the latest Dev version. Try updating

Is what I downloaded not the latest? I apologize if this seems rude, I genuinely am not sure, when I got it from the site I assumed it was

**Comment by Boomer-1** [comment]
Here's the latest version. You should run the dev versions, not the RC. Those are mainly for dev's who make addons.

**Comment by Kyrocakex** [comment]
> Here's the latest version. You should run the dev versions, not the RC. Those are mainly for dev's who make addons.
> 
> https://thebusybiscuit.github.io/builds/TheBusyBiscuit/Slimefun4/master/#971

Thank you for the quick responses! Sadly this did not work. I tried re-installing exotic gardens as well, making sure to get the latest file, though it doesn't seem to be working still, and has the same error message. (I followed the link you sent me as well and used that direct download) 

![image](https://user-images.githubusercontent.com/89564741/134459075-214d218f-daa0-418d-8d08-a689ec9be0b3.png)

Could it possibly be that my spigot just isn't updated properly, or may there be some other issue? The grindstone can't recognize multiple items, but wheat flour is the example I wanted to use.

**Comment by Boomer-1** [comment]
you could try updating spigot. there are 2 newer versions.  also can you show /pl

**Comment by Kyrocakex** [comment]
![image](https://user-images.githubusercontent.com/89564741/134611143-44f16ad6-6766-4f42-a72d-18ca88ee8db6.png)

So I found out what the issue is. The recipes are in the wrong place, wheat flour can't be created because it isn't recognized as wheat in this version (diamonds create wheat in the grindstone) and I've reinstalled / updated multiple times, with both the link you gave me and outside of it, what should I do? Is there a way to get around this? 

Also here's the PL: 
![image](https://user-images.githubusercontent.com/89564741/134611374-924be423-de47-4fda-8e6c-74602be7eed3.png)

**Comment by Boomer-1** [comment]
Like I've said, it works fine for me. The wheat must be in the upper left hand corner. are there any nbt tags on your wheat?

**Comment by Boomer-1** [comment]
also can you show the wheat flour unlocked in your guide?

**Comment by Kyrocakex** [comment]
Yes I have wheat unlocked (see attachment below) and there are no other tags, just straight up normal wheat. I unlocked everything prior to doing this, though it's the recipes themselves that are messed up for me for me. When I put diamonds in the grinder, it makes wheat, yet "wheat flour" is stationed as something that makes "Ice." 

I'm not entirely sure how to explain it better, so I recorded the process instead. You can find it here: https://youtu.be/SPODjiR3n6g

![2021-09-24_01 34 25](https://user-images.githubusercontent.com/89564741/134623490-76566a78-170c-4603-9323-ab62aac4db87.png)

**Comment by Boomer-1** [comment]
By the way, this isn't an Exotic Garden issue, never was. This should have been opened on the Slimefun github. I'm going to recommend you open this issue on the Slimefun bug tracker. I would include a link to this one to show what we've talked about so far. Here's where it should be reported. 

https://github.com/Slimefun/Slimefun4/issues

I still cannot replicate what you're seeing, but I did go into the grindstone in the guide and I see that I am missing the Eye of ender to convert into ender lumps.

![image](https://user-images.githubusercontent.com/75179680/134709968-b151fbf5-38a0-42c2-b673-2e21fb9ccabc.png)
